src/FirstSteps/RPG/Heroes/Hero.cs

```diff
@@ -15,7 +15,7 @@ public abstract class Hero : IAttacker
         protected int _health;
         public int Health { get { return _health; } }
         protected int _damage;
-        private int _coinsBag;
+        public int _coinsBag;
```

Comment:
It could be `protected` like for example `_strength`. https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/protected